central/convert/storagetov2/vuln_exception_service.go

```diff
@@ -0,0 +1,168 @@
+package storagetov2
+
+import (
+	"github.com/pkg/errors"
+	v2 "github.com/stackrox/rox/generated/api/v2"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/utils"
+)
+
+// VulnerabilityException converts *storage.VulnerabilityRequest to *v2.VulnerabilityException.
+func VulnerabilityException(inp *storage.VulnerabilityRequest) *v2.VulnerabilityException {
```

Comment:
`inp` seems like an odd name.   what is that short for?

Reply:
I was going for input. I think I will change it.